models/yang/sonic/sonic-acl.yang

```diff
@@ -0,0 +1,233 @@
+module sonic-acl {
+	namespace "http://github.com/Azure/sonic-acl";
+	prefix sacl;
+	yang-version 1.1;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	import sonic-port {
+		prefix prt;
```

Comment:
Prefix is used instead of module name. For example instead of using "sonic-vlan-interface", one can use "vif" as prefix. 

There is no proper guideline for deciding prefix name from module name. 

IETF YANG modules uses very short naming 
ietf-network-instance -> ni
ietf-interfaces -> if

Openconfig uses prefix similar to "oc-xxxxx" :
openconfig-network-instance --> "oc-netinst"
openconfig-rsvp-sr-ext --> "oc-sr-rsvp-ext".


IMO, we may not add "sonic-" as prefix in the prefix itself. We may follow IETF style short prefix. Please note that in SONiC YANG module we will not be using any external modules like OC or IETF YANG except ietf-inet-types.yang and ietf-yang-types.yang for standard data type, so there is no need to add "sonic-" in prefix. All data definitions are within SONiC only.


Example :
sonic-common --> cmn
sonic-acl ---> acl
sonic-port --> prt or port
sonic-interface -->if
sonic-vlan-interface -> vif
sonic-lag-interface --> lif
sonic-spanning-tree -->stp

Reply:
Are you going to change scommon & sacl to cmn & acl ?

I am not 100% certain, how community would perceive it. Let us go this way for now. 

Todo:
1) Fix scommon & sacl
2) Add this to guidelines doc please.

Reply:
**Fixed**